doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -38,7 +38,7 @@ Added the following features as supported:
 * Matter:
 
   * Matter 1.4.1: Matter commissioning using NFC tag can be officially certified now.
-  * :ref:`matter_manufacturer_specific_sample` sample: Dedicated sample, containing documentation and a preview :ref:`PC tool <ug_nrfconnect_manufacturer_cluster_editor_tool>`, facilitating creation and modification of manufacturer specific clusters.
+  * :ref:`matter_manufacturer_specific_sample` sample: Dedicated sample, containing documentation and a preview :ref:`PC app <ug_matter_gs_tools_matter_cluster_editor>`, facilitating creation and modification of manufacturer specific clusters.
```

Comment:
Do we need to edit the previous release notes?

Reply:
Not in the already released bundles. This just fixes the link for those who click it from the 3.1.0 or future bundles. The old release bundles will still point to the page we're removing here.